cli/JSXV4.md

```diff
@@ -98,3 +126,15 @@ function has the name of the enclosing module/file.
 
 ReactDOMRe.createElement(ReasonReact.fragment, [comp1, comp2, comp3])
 ```
+
+**File-level config**
+
+The top-level attribute `@@jsxConfig` is used to update the jsx config for the rest of the file (or until the next config update). Only the values mentioned are updated, the others are left unchanged.
+
+```rescript
+@@jsxConfig({version: 4, mode: "automatic"})
+// The jsx config is updated for the rest of the file.
```

Comment:
this also seems old -- needs a bit more explanation of what is possible

Reply:
add some explanations there